static/app/views/replays/detail/issueList.tsx

```diff
@@ -59,8 +59,17 @@ function IssueList({projectId, replayId}: Props) {
         }
       >
         {issues
-          // prioritize the replay issues first
-          .sort(a => (a.project.id === projectId ? -1 : 1))
+          // prioritize the replay issues for the project first, followed by first_seen
+          .sort((a, b) => {
+            if (a.project.id === projectId) {
+              if (a.project.id === b.project.id) {
+                return new Date(a.firstSeen).getTime() - new Date(b.firstSeen).getTime();
+              }
+
+              return -1;
+            }
+            return 1;
+          })
```

Comment:
Probably easier to read as
```suggestion
          .sort((a, b) => {
            if (a.project.id !== projectId) {
              return 1;
            }
            
            if (a.project.id === b.project.id) {
              return new Date(a.firstSeen).getTime() - new Date(b.firstSeen).getTime();
            }

            return -1;
          })
```